lib/uring/uring.mli

```diff
@@ -241,34 +345,76 @@ module Statx : sig
   ]
 
   val pp_kind : kind Fmt.t
+  (** [pp_kind kind] formats a human readable [kind] *)
 
   val create : unit -> t
   (** Use [create] to make a statx result buffer to pass to {! statx}. *)
 
   module Flags : sig
     include FLAGS
     
-    val empty : t
     val empty_path : t
+    (** [empty_path] signals that if the pathname is an empty string
+        then operate on the file referred to by the [fd] (which can
+        refer to any type of file).
+        If [fd] is not specified then the call operates on the current
+        working directory. *)
+
     val no_automount : t
+    (** [no_automount] signals that statx should not automount the basename
+        component of the path if it is an automount point.
+        This can be used in tools that scan directories to prevent
+        mass-automounting of a directory of automount points. *)
+
     val symlink_nofollow : t
+    (** [symlink_nofollow] signals that if the path is a symbolic link,
+        then return return information about the link itself. *)
```

Comment:
```suggestion
        then return information about the link itself. *)
```

Reply:
fixed in f9769b8d1cfe14a619182467dec21aef1965b2f5